content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,81 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+### Features
+
+#### Replicate data remotely
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+- https://github.com/influxdata/influxdb/commit/5a1e375): Build: allow to build on FreeBSD
```

Comment:
does this mean we allow installs of 2.2 on FreeBSD? do we need to update install instructions for this?

Reply:
I think this just enables the build process.  We do not plan to support FreeBSD.  So, I think we can skip the docs for this.

Reply:
Removed:

- Build: allow to build on FreeBSD